From c2b681e39421ae5b4fea917b3c3ed69bd1020b6c Mon Sep 17 00:00:00 2001
From: Wenwen Fu <wenwfu@qti.qualcomm.com>
Date: Tue, 8 Sep 2026 22:08:57 +0800
Subject: [PATCH] busybox: enable findfs for initramfs LABEL support

Use the BusyBox findfs applet for initramfs-framework root=LABEL=
resolution and drop the util-linux-findfs runtime dependency. This keeps
the root-only udev trigger implementation lightweight and removes the
util-linux signature exception.

Signed-off-by: Wenwen Fu <wenwfu@qti.qualcomm.com>
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Upstream-Status: Inappropriate [qcom-distro-specific adaptation of https://github.com/openembedded/openembedded-core/commit/c2b681e39421ae5b4fea917b3c3ed69bd1020b6c]
---
 meta/recipes-core/busybox/busybox/defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/defconfig b/defconfig
index 1f4b5e3d1e..a282370a8f 100644
--- a/defconfig
+++ b/defconfig
@@ -634,7 +634,7 @@ CONFIG_FEATURE_FDISK_WRITABLE=y
 # CONFIG_FEATURE_OSF_LABEL is not set
 # CONFIG_FEATURE_GPT_LABEL is not set
 # CONFIG_FEATURE_FDISK_ADVANCED is not set
-# CONFIG_FINDFS is not set
+CONFIG_FINDFS=y
 CONFIG_FLOCK=y
 # CONFIG_FDFLUSH is not set
 # CONFIG_FREERAMDISK is not set
-- 
2.43.0

